volatility3/framework/plugins/linux/sockscan.py

```diff
@@ -0,0 +1,354 @@
+# This file is Copyright 2024 Volatility Foundation and licensed under the Volatility Software License 1.0
+# which is available at https://www.volatilityfoundation.org/license/vsl-v1.0
+#
+
+import logging
+import struct
+from typing import List, Set
+
+from volatility3.framework import exceptions, constants
+from volatility3.framework.renderers import TreeGrid, NotAvailableValue, format_hints
+from volatility3.framework.configuration import requirements
+from volatility3.framework.interfaces import plugins
+from volatility3.framework.symbols import linux
+from volatility3.plugins.linux import sockstat
+from volatility3.framework import symbols
+from volatility3.framework import symbols, constants
```

Comment:
I think this code is duplicate.
```suggestion
from volatility3.framework import symbols, constants
```

Reply:
Yup, totally missed that. I'll get it tidied up.